packages/datatrak-web/src/features/MobileAppPrompt.tsx

```diff
@@ -8,6 +8,7 @@ import React, { useState } from 'react';
 import { useIsMobile } from '../utils';
 import styled from 'styled-components';
```

Comment:
I know it wasn't from your changes, but just need to reorder some of these imports